backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -154,21 +166,38 @@
     // Note: This method uses a stateless session
```

Comment:
I wonder if we should create and use an annotation like `@StatelessSession` (or a better name) with `RetentionPolicy.SOURCE`. It would be more visible than a comment in my IntelliJ configured with the dark theme 😄

Reply:
That would be nice, would be even better if we could detect mixed usage (at least at compile or test time)

Reply:
That said, we could do that in a separate PR

Reply:
https://issues.redhat.com/browse/NOTIF-445